addons/crm/crm_lead.py

```diff
@@ -730,28 +730,35 @@ def convert_opportunity(self, cr, uid, ids, partner_id, user_ids=False, section_
 
         return True
 
+    @api.one
```

Comment:
I don't know, but it works fine. Less lines to rewrite in the future. I could have written longer and older code, but did not make sense to me. Api is there to compatibilize old and new styles, I don't know why not use it.